en/brands.ftl

```diff
@@ -42,6 +42,7 @@
 -brand-name-firefox-nightly = Firefox Nightly
 -brand-name-firefox-reality = Firefox Reality
 -brand-name-firefox-daylight = Firefox Daylight
+-brand-name-firefox-addons = Firefox Add-ons
```

Comment:
Note: this shouldn't be here, we've removed it in bedrock just waiting for the updates to cross over.